runtime-config-linux.md

```diff
@@ -201,6 +201,8 @@ The following parameters can be specified to setup the controller:
 
 * **`kernel`** *(uint64, optional)* - sets hard limit for kernel memory
 
+* **`kernelTCP`** *(uint64, optional)* - sets hard limit for kernel memory in tcp using
```

Comment:
Err, actually, he's adjusting docs for other settings.  I'll file a PR
for this on.